mypy_django_plugin/transformers/managers.py

```diff
@@ -0,0 +1,153 @@
+from typing import Tuple, List
+
+from mypy.nodes import (
+    GDEF, Argument, FuncDef, MemberExpr, RefExpr, NameExpr, PlaceholderNode,
+    StrExpr, SymbolTableNode, TypeInfo, Var,
+)
+from mypy.plugin import ClassDefContext
+from mypy.types import AnyType, CallableType, Instance, TypeOfAny
+
+from mypy_django_plugin.lib import fullnames, helpers
+from mypy.plugins.common import add_method_to_class
+
+from mypy.types import Type as MypyType
+
+
+def build_unannotated_method_args(method_node: FuncDef) -> Tuple[List[Argument], MypyType]:
+    prepared_arguments = []
+    for argument in method_node.arguments[1:]:
+        argument.type_annotation = AnyType(TypeOfAny.unannotated)
+        prepared_arguments.append(argument)
+    return_type = AnyType(TypeOfAny.unannotated)
+    return prepared_arguments, return_type
+
+
+class ManagerCallback(helpers.DynamicClassFromMethodCallback):
+
+    def copy_method_to_another_class(
+            self,
+            ctx: ClassDefContext,
+            self_type: Instance,
+            new_method_name: str,
+            method_node: FuncDef) -> None:
+        if method_node.type is None:
+            if self.defer_till_next_iteration():
+                return
+
+            arguments, return_type = build_unannotated_method_args(method_node)
+            add_method_to_class(
+                    ctx.api,
+                    ctx.cls,
+                    new_method_name,
+                    args=arguments,
+                    return_type=return_type,
+                    self_type=self_type)
+            return
+
+        method_type = method_node.type
+        if not isinstance(method_type, CallableType):
+            if not self.semanal_api.final_iteration:
```

Comment:
`defer_till_next_iteration` too?

Reply:
@mkurnikov will `if not isinstance(method_type, CallableType) and not self.defer_till_next_iteration()` work? I'm not 100% sure if I get how deferring works

Reply:
If it's `not isinstance(CallableType)`, which type it is? Placeholder?

Reply:
It might be ` Overloaded`, `CallableType` or `None` actually